README.md

```diff
@@ -161,6 +161,7 @@ SPACESHIP_PROMPT_ORDER=(
   golang        # Go section
   php           # PHP section
   rust          # Rust section
+  stack         # Haskell Stack section
```

Comment:
Please, add Haskell to list of features at the beginning of the readme. Something like:

> Current Haskell version, through stack (`λ`).